std/regex/package.d

```diff
@@ -1248,18 +1235,21 @@ public @trusted void replaceAllInto(alias fun, Sink, R, RegEx)
     replaceMatchesInto!fun(sink, input, matchAll(input, re));
 }
 
-// a bit of examples
+///
 @system unittest
 {
-    //swap all 3 letter words and bring it back
-    string text = "How are you doing?";
-    auto sink = appender!(char[])();
-    replaceAllInto!(cap => retro(cap[0]))(sink, text, regex(`\b\w{3}\b`));
-    auto swapped = sink.data.dup; // make a copy explicitly
-    assert(swapped == "woH era uoy doing?");
-    sink.clear();
-    replaceAllInto!(cap => retro(cap[0]))(sink, swapped, regex(`\b\w{3}\b`));
-    assert(sink.data == text);
+    // insert comma as thousands delimiter in fifty randomly produced big numbers
+    import std.array, std.random, std.conv, std.range;
+    static re = ctRegex!(`(?<=\d)(?=(\d\d\d)+\b)`, "g");
```

Comment:
I have to say @quickfur is into something here. The thing about ctRegex being faster comes at a cost of compile-time and we absolutely want faster test runs.

So let's make it plain regex and I'll merge it then.